benchmark/test_reduction_perf.py

```diff
@@ -288,6 +288,58 @@ def test_perf_vector_norm():
     bench.run()
 
 
+def test_perf_slice_scatter():
```

Comment:
as we discussed, performance tests are not needed temporarily. we could supply them till the performance is competitive with torch eager.